src/server_manager/web_app/browser_main.ts

```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-(window as any).trustCertificate = (fingerprint: string) => {
-  console.log(`Requested to trust certificate with fingerprint ${fingerprint}`);
+(window as any).trustCertificate = (anchor: HostAnchor) => {
+  console.log(`Requested to trust certificate: ${anchor}`);
```

Comment:
You might need to call JSON.stringfy here.

Reply:
No, we can pass JSON objects across the boundary.

Reply:
I meant for the log call. Is that properly serializing the object?

Reply:
(obsolete)